backend/src/tests/test_registries/e2e_tests.py

```diff
@@ -97,3 +102,705 @@ def test_delete(self, organization, api_client):
+
+    @pytest.mark.parametrize("field", ("unit", "abbreviation", "okei_code"))
+    def test_partial_update(self, field, organization, api_client):
+        unit = baker.make("registries.MeasurementUnit", organization=organization)
+
+        new_value = getattr(unit, field)[:5] + "_new" if getattr(unit, field) else 999
+        response = api_client.patch(
+            f"{self.endpoint}{unit.id}/", data={field: new_value}, format="json"
+        )
+
+        assert response.status_code == 200
+        assert simplejson.loads(response.content)[field] == new_value
+
+    def test_delete(self, organization, api_client):
+        unit = baker.make("registries.MeasurementUnit", organization=organization)
+
+        response = api_client.delete(f"{self.endpoint}{unit.id}/")
+
+        assert response.status_code == 204
+        assert organization.measurement_units.all().count() == 0
+
+
+class TestWorkplacesEndpoints:
+    endpoint = "/api/v1/organizations/1/registries/workplaces/"
+
+    def test_list(self, organization, api_client):
+        baker.make("registries.Workplace", organization=organization, _quantity=3)
+
+        response = api_client.get(self.endpoint)
+
+        assert response.status_code == 200
+        assert simplejson.loads(response.content)["count"] == 3
+
+    def test_create(self, organization, api_client):
+        workplace = WorkplaceFactory.build(organization=organization)
+        expected_json = {
+            "name": workplace.name,
+            "description": workplace.description,
+            "color": workplace.color,
+        }
+
+        response = api_client.post(self.endpoint, data=expected_json, format="json")
+
+        assert response.status_code == 201
+
+        response_data = simplejson.loads(response.content)
+
+        assert "id" in response_data
+        assert response_data["name"] == expected_json["name"]
+        assert response_data["description"] == expected_json["description"]
+        assert response_data["color"] == expected_json["color"]
+        assert "icon" in response_data
+
+    def test_retrieve(self, organization, api_client):
+        workplace = baker.make("registries.Workplace", organization=organization)
+
+        expected_json = {
+            "id": workplace.id,
+            "name": workplace.name,
+            "description": workplace.description,
+            "color": workplace.color,
+            "icon": workplace.icon,
+        }
+
+        response = api_client.get(f"{self.endpoint}{workplace.id}/")
+        response_data = simplejson.loads(response.content)
+
+        for field in ["created_at", "updated_at"]:
+            del response_data[field]
+
+        assert response.status_code == 200
+        assert response_data == expected_json
+
+    def test_update(self, organization, api_client):
+        old_workplace = baker.make("registries.Workplace", organization=organization)
+        new_workplace = WorkplaceFactory.build(organization=organization)
```

Comment:
почему бы, как и везде, не использовать baker.prepare? Если какие-то специфичные поля не проходят, их можно прописать вручную

```suggestion
        new_workplace = baker.prepare(
            "registries.Workplace", organization=organization, color="#999"
        )
```